configure

```diff
@@ -747,6 +747,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
All of these `runstatedir`-related changes get overwritten if you re-run `autoconf`

Reply:
Although I installed the older autoconf-2.69 on my system to minimize changes to the generated files, Gentoo does add a few patches which cause those differences. I'm fairly sure I checked that the extra changes didn't materially impact the installation. You can leave them in or regenerate the file on your reference platform.